src/ontology/mondo-edit.obo

```diff
@@ -432996,6 +432997,20 @@ xref: ICD10:F16.183 {source="MONDO:relatedTo"}
 is_a: MONDO:0024417 {source="https://orcid.org/0000-0001-5208-3432"} ! perceptual disorders
 created_by: http://orcid.org/0000-0001-5208-3432
 
+[Term]
+id: MONDO:0100127
+name: Imerslund-Grasbeck syndrome 2
+def: "An autosomal recessive disorder characterized by onset of megaloblastic anemia associated with decreased serum vitamin B12 (cobalamin, Cbl) in infancy or early childhood. Low molecular weight (LMW) proteinuria is frequently present, but usually occurs later and is usually mild or subclinical. Patients often present with vague symptoms, including failure to thrive, loss of appetite, fatigue, lethargy, and/or recurrent infections." [OMIM:618882]
+synonym: "megaloblastic anemia, norwegian type" EXACT [OMIM:618882]
+xref: OMIM:618882 {source="MONDO:equivalentTo"}
+is_a: MONDO:0001700 {source="OMIM:618882"} ! megaloblastic anemia (disease)
```

Comment:
hmm, feels like is-a overloaading and doesn't strictly come from omim. Seems like the isas for the generic type came from ordo, you coulld cite those here.. or review whether this is warraanted aat all, some of the parents may be in the obs candidates

Reply:
I added the parent class Imerslund-Grasbeck syndrome and moved Type 1 and Type 2 as children.

The parent megaloblastic anemia comes from NCIT:C131677 and DC-OMIM:261100

Reply:
Let me know if further changes are needed